modules/cluster-logging-rn-5.7.1.adoc

```diff
@@ -0,0 +1,27 @@
+
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-7-1_{context}"]
+= Logging 5.7.1
+This release includes: link:https://access.redhat.com/errata/RHBA-2023:3197[OpenShift Logging Bug Fix Release 5.7.1].
+
+[id="openshift-logging-5-7-1-bug-fixes"]
+== Bug fixes
+* Before this update, the presence of numerous noisy messages within Cluster Logging Operator pod logs caused reduced log readability and increased difficulty in identifying important system events. With this update, the issue is resolved by significantly reducing the noisy messages within Cluster Logging Operator pod logs, resulting in improved log readability and enhanced visibility of critical system events. (link:https://issues.redhat.com/browse/LOG-3482[LOG-3482])
```

Comment:
```suggestion
* Before this update, the presence of numerous noisy messages within the Cluster Logging Operator pod logs caused reduced log readability, and increased difficulty in identifying important system events. With this update, the issue is resolved by significantly reducing the noisy messages within Cluster Logging Operator pod logs. (link:https://issues.redhat.com/browse/LOG-3482[LOG-3482])
```

suggested edit for readability - added a comma, and I don't think the last line needs to be there since it's already mentioned previously as the result of having too many messages

Reply:
Have you considered also adding an attribute for Cluster Logging Operator? Not important for this PR since it's maybe out of scope, but worth considering opening a follow up issue IMO.